releasenotes/notes/fix-barrier-before-final-measurements-loose-1849282c11fc5eb0.yaml

```diff
@@ -3,6 +3,6 @@ fixes:
   - |
     Fixed the :class:`.BarrierBeforeFinalMeasurements` transpiler pass when there
     are conditions on loose :class:`.Clbit`\ s immediately before the final measurement
-    layer.  Previously, this would fail claiming that the bit was not present
+    layer. Previously, this would fail claiming that the bit was not present
```

Comment:
This is usually a giveaway that I wrote the release note lol.  It won't actually matter at all because Sphinx normalises spaces like this.

Reply:
ok good to know Sphinx fixes it 👍🏻